Header header

string[] name

#message from Watcom Tablet PC
float64[] tip_position
float64[] tilt_angle
float64   pressure

#message from texscan
int64[] thumb1_f
int64[] thumb2_f
int64[] index1_f
int64[] index2_f
int64[] index3_f
int64[] middle1_f
int64[] middle2_f
int64[] middle3_f
int64[] ring1_f
int64[] ring2_f
int64[] ring3_f
int64[] pinky1_f
int64[] pinky2_f
int64[] pinky3_f
int64[] palm1_f
int64[] palm2_f
int64[] palm31_f
int64[] palm32_f
int64[] thumb1_s
int64[] thumb2_s
int64[] index1_s
int64[] index2_s
int64[] index3_s
int64[] middle1_s
int64[] middle2_s
int64[] middle3_s
int64[] ring1_s
int64[] ring2_s
int64[] ring3_s
int64[] pinky1_s
int64[] pinky2_s
int64[] pinky3_s
int64[] palm1_s
int64[] palm2_s
int64[] palm31_s
int64[] palm32_s
float64   thumb1_f_avg
float64   thumb2_f_avg
float64   index1_f_avg
float64   index2_f_avg
float64   index3_f_avg
float64   middle1_f_avg
float64   middle2_f_avg
float64   middle3_f_avg
float64   ring1_f_avg
float64   ring2_f_avg
float64   ring3_f_avg
float64   pinky1_f_avg
float64   pinky2_f_avg
float64   pinky3_f_avg
float64   palm1_f_avg
float64   palm2_f_avg
float64   palm31_f_avg
float64   palm32_f_avg
float64   thumb1_s_avg
float64   thumb2_s_avg
float64   index1_s_avg
float64   index2_s_avg
float64   index3_s_avg
float64   middle1_s_avg
float64   middle2_s_avg
float64   middle3_s_avg
float64   ring1_s_avg
float64   ring2_s_avg
float64   ring3_s_avg
float64   pinky1_s_avg
float64   pinky2_s_avg
float64   pinky3_s_avg
float64   palm1_s_avg
float64   palm2_s_avg
float64 palm31_s_avg
float64 palm32_s_avg

#message from cyber glove
int64[] finger_pos

#message from ATI F/T sensors
float64   tip_force_x
float64   tip_force_y
float64   tip_force_z
float64   tip_torque_x
float64   tip_torque_y
float64   tip_torque_z

#message from OptiTracker
geometry_msgs/Transform     obj_pose 
